indico/web/templates/forms/_person_link_widget_base.html

```diff
@@ -15,12 +15,13 @@
 {% endmacro %}
 
 {% block html %}
-    {% set custom_display_order = field._value()|selectattr('displayOrder')|list|length > 0 %}
+    {% set list_in_custom_order = (field._value()|selectattr('displayOrder')|list|length > 0) %}
```

Comment:
It's not needed. Probably left over from a previous version where I was joining that clause with something else.